wbpack.config.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
```

Comment:
- [ ] Unfortunately, the npm scripts are not running properly on this project because this file name is misspelled. Kindly rename this file to `webpack.config.js` so the scripts can run.